Userland/Games/BrickGame/BrickGame.cpp

```diff
@@ -0,0 +1,178 @@
+/*
+ * Copyright (c) 2018-2020, Andreas Kling <kling@serenityos.org>
```

Comment:
You aren't Andreas! Feel free to replace this with your own, or keep the one below -- but not both :^)